app/assets/stylesheets/pages/articles/article-detail/_share.scss

```diff
@@ -0,0 +1,138 @@
+.share{
+	margin-top: 42px;
+
+	&__txt{
+		color: $textColorSecondary;
+		text-transform: capitalize;
+	}
+
+	&__sosmed-wrapper{
+		display: inline-block;
+
+		@media only screen and (max-width: 375px){
+			display: block;
+		}
+	}
+
+	&__sosmed-link{
+		display: inline-block;
+		background-color: $colorPrimary;
+		padding: 4px;
+		border-radius: 8px;
+		margin: 3px 10px 0 0;
```

Comment:
if you want to raise the gap, try  give margin top to the parent container not the tag a

Reply:
yet